pkg/tsdb/mssql/mssql.go

```diff
@@ -139,14 +139,11 @@ func generateConnectionString(dsInfo sqleng.DataSourceInfo) (string, error) {
 		}
 	}
 
-	args := []interface{}{
-		"url", dsInfo.URL, "host", addr.Host,
-	}
-	if addr.Port != "0" {
-		args = append(args, "port", addr.Port)
-	}
-
-	logger.Debug("Generating connection string", args...)
+	logger.Info("Generating connection string from", "args", dsInfo)
```

Comment:
Not sure this change is for the better using `dsInfo` as args. Please change back to log the URL, host and port as args using debug level.

Reply:
fixed